.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
bro, you accidentally pushed all of the `.idea` folder with various IDE config files - you need to add them to `.gitignore` and [delete them from git cache](https://stackoverflow.com/questions/13541615/how-to-remove-files-that-are-listed-in-the-gitignore-but-still-on-the-repositor#:~:text=gitignore%20are%20not%20being%20tracked,unix)%20will%20give%20you%20help.)